web/src/elements/rbac/ObjectPermissionsPage.ts

```diff
@@ -76,4 +86,78 @@ export class ObjectPermissionPage extends AKElement {
                 </section>
             </ak-tabs>`;
     }
+
+    renderCoreUser() {
```

Comment:
@BeryJu broke out `renderControlButtons` above in the UserView page, and it's a good practice; break out big pieces of functionality, especially when they're rendered conditionally, and name them.

Reply:
Two reasons, actually: the first is that [Typescript generally recommends against it](https://www.typescriptlang.org/docs/handbook/2/functions.html), as inference of return type is very good and the only reason to do it is if you don't trust the function body itself to generate the correct return type, and the other is that Lit's re-render algorithm is smarter with `nothing` rather than `html\`\`` , but doesn't conform to TemplateResult without a lot of fussiness.